test/t/test_sudo.py

```diff
@@ -81,3 +82,13 @@ def test_11(self, bash, part_full_group):
         part, _ = part_full_group
         completion = assert_complete(bash, "sudo chown foo:bar:%s" % part)
         assert not completion
+
+    def test_12(self, bash):
+        completion = assert_complete(bash, 'sudo "/tmp/aaa bbb" ')
+        output = assert_bash_exec(
+            bash, "complete -p aaa || true", want_output=True
+        )
+        assert output.endswith(
+            "bash: complete: aaa: no completion specification"
+        )
```

Comment:
This test depends on the details of the error messages that might change in future Bash. I feel that we can just test that `complete -p aaa` fails. We don't set `errexit` (`set -e`) while testing. I suggest something like the following simpler test though I didn't test it:

```suggestion
        assert_bash_exec(bash, "! complete -p aaa", want_output=None)
```

What do you think?